문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,13 @@
+function solution(s) {
+    var answer = false;
+    let counts = {
+        p: 0,
+        y: 0,
+    };
+    for (var i = 0; i <= s.length; i++) {
+        s[i] === "P" || s[i] === "p" ? (counts["p"] += 1) : counts;
+        s[i] === "Y" || s[i] === "y" ? (counts["y"] += 1) : counts;
```

Comment:
삼항연산자보다 &&연산자를 사용하는게 좋을거 같아요
```suggestion
        (s[i] === "P" || s[i] === "p") && (counts["p"] += 1);
        (s[i] === "Y" || s[i] === "y") && (counts["y"] += 1);
```